### ISSUE TITLE
Feature request: Snooze for 1 hour/1 day links

### ISSUE THREAD

**Issue by erikgeurts** [leggett/simplify#754]
I use the Snooze feature in Gmail a lot, and it's an important part of my workflow.

Before Gmail introduced Snooze, I used something similar in another extension. This particular extension had a helpful variation, which was the ability to snooze an email for 1 hour from "now", or for 1 day. It would automatically and dynamically snooze the email from the current date and time.

I was wondering if it might be possible for Simplify to add something like this to the existing snooze dropdown.

**Comment by leggett** [comment]
Yeah, I have a backlog of small improvements to Snooze like this.

**Comment by irnoham** [comment]
+1
I would love this.
